modules/ossm-config-openshift-monitoring-only.adoc

```diff
@@ -0,0 +1,99 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/metrics/ossm-metrics-assembly.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ossm-config-openshift-monitoring-only_{context}"]
+= Configuring OpenShift Monitoring with Service Mesh
+:context: ossm-config-openshift-monitoring-only
+
+//Need some kind of intro
+//Possible file name may change
+//Possible assembly file may change
+//Assemblies, topic map info needs to be worked out still for 3.0, now that docs has a better idea of what is to be included for 3.0 TP1.
+
+//FIX THE STYLE THINGS ONCE DEV HAS REVIEWED/APPROVED THE CONTENT. First priority is ensuring the steps are in the correct order, and are accurate for 3.0 TP1.
+
+The following steps show how to integrate {SMProductName} with user-workload monitoring.
+
+.Prerequisite
+
+ * Monitoring for user-defined projects is enabled. See "Enabling monitoring for user-defined projects".
```

Comment:
Can 'See "Enabled monitoring for user-defined  projects"' be a link? Otherwise you have to scroll down to the bottom of the page to find the link.

Reply:
Unfortunately, no. For style and technical reasons, they must go in an "Additional resources" section.